### PR TITLE
MONA-137 Fix Linting and console errors.

### PR DESCRIPTION
**Pull request by acsearles**
Ok this one should be a little more clear as to what I actually changed.

### CODE REVIEW

**Comment by acsearles** [comment]
I also renamed a lot of files in here to so of course disregard that stuff. Oh and the dist files of course.

File: lib/dist/iui-module.js [hunk 1]
```diff
@@ -335,74 +323,103 @@ var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol
...
-})(window.app);
+})();
 'use strict';
 
-(function (app) {
+(function (ng) {
   'use strict';
 
-  app.directive('mfCheckBoxList', ['$filter', function ($filter) {
+  angular.module('iuiForm').directive('mfAccessDropDown', ['DropDownListItemsSvc', '$log', '_', 'session', function (ddSvc, log, _, session) {
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: why are we including mfAccessDropDown?

Comment (acsearles) [inline lib/dist/iui-module.js]: Took it out for a later commit. I'm saving it so you don't have to code review a ton of messed up diffs.

**Comment by acsearles** [comment]
@mstone6769 is there anything on here I need to take care of?

**Comment by mstone6769** [comment]
@acsearles I think this is as much as we can do for now...some time will need to be taken to fix and remove Empower dependencies from these

**Pull request merged**